### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by cristianvasquez** [gavvvr/obsidian-imgur-plugin#2] [General]
Thanks for the plugin!

No more polluting the git history with binaries, and also sharing between vaults :)

**Comment by gavvvr** [comment]
Hi @cristianvasquez. Glad that you like it :) I usually try to avoid binaries in git repos at any cost. Even if they are not supposed to change.
Do you use https://github.com/denolehov/obsidian-git or manually manage git repo to backup your notes?

**Comment by cristianvasquez** [comment]
I use obsidian-git to automatically do the commits every 10 minutes, but I choose when to do the push. I like it that way because I can have granular information about the evolution of my notes. Perhaps in the future, there will be a tool that uses this git information to enhance productivity :).   is is also cool to watch your repo using https://gource.io/
